dialogflow/stopwatch/README.md

```diff
@@ -1,42 +1,70 @@
+- [Create a Service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) with the following IAM role: Dialogflow API Client. Example name: `dialogflow-client`
+- For your "App Engine Default Service Account" add the following role "Service Account Token Creator" Role.
+- To use Sentiment Analysis you need to [Enable beta features](https://cloud.google.com/dialogflow-enterprise/docs/sentiment#enable_beta_features) 
+- To use Text-to-Speech you need to [Enable beta features](https://cloud.google.com/dialogflow-enterprise/docs/detect-intent-tts#enable_beta_features)
+- To use Knowledge Connectors you need to [Enable beta features](https://cloud.google.com/dialogflow-enterprise/docs/knowledge-connectors#enable_beta_features) 
 
-## Enable Dialogflow
-If you have not already done so, [enable Dialogflow for your project](https://cloud.google.com/dialogflow-enterprise/docs/quickstart). Scripts related to the quickstart are in the SETUP-SERVICE directory.
 
-## Running the app
-- In the [Google Cloud Console](https://console.cloud.google.com), use the APIs&Services->Library menu item to find and enable the Dialogflow API.
-- Also using the Credentials item in the Google Cloud Console, create a Service Account key. Use the popup to create a key for a "New service account", name it "stopwatch" (this is arbitrary), and give it the role of `Dialogflow API Admin`. Choose a key type of "JSON" and press the "Create" button to download a key file. As a convenience, rename this file to `credentials.json`.
+### Setup the app
 - Clone this repository and `cd` into this directory.
-- Copy your `credentials.json` over the skeletal version in this directory. It is used by the scripts in `SETUP_SERVICE` and by the sample application. 
-- Be sure that you have gone through the steps in the [Dialogflow quickstart](https://cloud.google.com/dialogflow-enterprise/docs/quickstart) to create and configure your stopwatch agent. Helper scripts for this are in the `SETUP_SERVICE` directory.
-- Run ./INSTALL-COCOAPODS to install app dependencies. When it finishes, it will open the Stopwatch workspace in Xcode. Since we are using Cocoapods, be sure to open the workspace and not Stopwatch.xcodeproj.
-- Replace `your-project-identifier` in `StopwatchService.swift` with the identifier of your Google Cloud project.
+- Run `./INSTALL-COCOAPODS` to install app dependencies. When it finishes, it will open the Stopwatch workspace in Xcode. Since we are using Cocoapods, be sure to open the workspace and not Stopwatch.xcodeproj.
+- Replace `your-project-identifier` in `ApplicationConstants.swift` with the identifier of your Google Cloud project.
+
+###  Setup Firebase on the application:
+
+- Complete the steps for [Add Firebase to your app](https://firebase.google.com/docs/ios/setup#add_firebase_to_your_app). Note: No need to complete any other sections, they are already done. 
```

Comment:
Add the `iOS bundle ID` as like an additional inside bullet
```
- Complete ...
   - Use `iOS bundle ID` as `com.sample.cce`
```
Not sure what `cce` means, might be worth change to `com.sample.dialogflow`

Reply:
Last thing, then LGTM :) 🎉

Reply:
Done, Thanks :)